boorong/description_old.md

```diff
@@ -0,0 +1,149 @@
+# Boorong
```

Comment:
Please avoid adding previous files versions unless strictly necessary. Git is already taking care of history for us!
If some info will be lost, then move it into a doc/ subdirectory.